packages/expo-random/ios/RandomModule.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+import Security
+import ExpoModulesCore
+
+public class RandomModule: Module {
+  public func definition() -> ModuleDefinition {
+    name("ExpoRandom")
+
+    function("getRandomBase64StringAsync", getRandomBase64String)
+
+    function("getRandomBase64String", getRandomBase64String)
+      .runSynchronously()
+  }
+}
+
+private func getRandomBase64String(length: Int) throws -> String {
+  var bytes = [UInt8](repeating: 0, count: length)
+  let status = SecRandomCopyBytes(kSecRandomDefault, length, &bytes)
+
+  guard status == errSecSuccess else {
+    throw FailedGeneratingRandomBytesException(status)
+  }
+  return Data(bytes).base64EncodedString()
+}
+
+private class FailedGeneratingRandomBytesException: GenericException<OSStatus> {
```

Comment:
Interesting. Maybe add a comment about what `GenericException<OSStatus>` is and its importance in regards to rejections.

Reply:
This is something from the core, it has some doc comments there.
I'm going to add the exceptions concept to the docs as well, but there are still some missing parts that I'd like to implement first.